Test Files/accountsettingtest.java

```diff
@@ -0,0 +1,61 @@
+import static org.mockito.Mockito.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
```

Comment:
Ensure all imports are added to dependecies once maven has been successfully integrated